app/views/layouts/_navbar.html.haml

```diff
@@ -10,7 +10,7 @@
             Home
             %span.sr-only (current)
         %li.nav-item.active
-          %a.nav-link{:href => "#"}
+          %a.nav-link{:href => "/pages/about"}
```

Comment:
Once you set routes for the about method run `rails routes` and use about path from there.